redis/commands/core.py

```diff
@@ -2224,6 +2224,39 @@ def sort(
         options = {"groups": len(get) if groups else None}
         return self.execute_command("SORT", *pieces, **options)
 
+    def sort_ro(
+        self,
+        key: str,
+        start: Optional[int] = None,
+        num: Optional[int] = None,
+        by: Optional[str] = None,
+        get: Optional[List[str]] = None,
+        desc: bool = False,
+        alpha: bool = False,
+    ) -> list:
+        """
+        Returns the elements contained in the list, set or sorted set at key.
+        (read-only variant of the SORT command)
+
+        ``start`` and ``num`` allow for paging through the sorted data
```

Comment:
Where do start and num come from? I dont't see them on the docs page. I see a LIMIT in which case we can align with the command.

Reply:
I did it this way to stay consistent with 'sort', I can change both but I don't think it's a good idea